web/thesauruses/c/io.json

```diff
@@ -0,0 +1,97 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "io"
+    },
+    "concepts": {
+        "write_line": {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
+            "name": "Write a line to the console"
+        },
+        "write_line_with_new_line" : {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
+            "name": "Write a line to the console with a return character"  
+        },
+        "read_line": {
+        "code": "scanf(“Format Specifier”, &var1, &var2, …., &varn);",
+        "name": "Read a line of user input from the console"
```

Comment:
I've done C++ so I know what the `&`s are for. Others newer to C/C++ may not. Would you add a `"comment": "..."` line that maybe explains why you need the `&` for read line but you don't need it for write line?

Reply:
added comment